### PR TITLE
Add the TSC to the contributing file

### PR DESCRIPTION
**Pull request by directionless**
This adds a section on the TSC to the contributing file.

It also reflows things.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,151 +1,192 @@
...
-development with the core team. You are welcome to join. Office hours are announced on our Slack on
-the `#officehours` channel.
+All contributions are submitted via pull requests (PRs) open against
+the osquery's [master](https://github.com/osquery/osquery/tree/master)
+branch on GitHub. After being reviewed by the _core team_ and tested
+by CI, if all is well, they will be pushed to master and the
+corresponding PR closed.
+
+You can see who the _core team_ is by viewing the [team
+page](https://github.com/orgs/osquery/teams) on the osquery GitHub
```
Comment (sharvilshah) [inline CONTRIBUTING.md]: I think this is not viewable to people outside of the teams :)

Comment (directionless) [reply CONTRIBUTING.md]: Hrm. That link should be https://github.com/orgs/osquery/teams/technical-steering-committee and I'm not sure. It's flagged as a visible team

Comment (sharvilshah) [reply CONTRIBUTING.md]: That is a 404 as well

Comment (directionless) [reply CONTRIBUTING.md]: Oh, I see now. This is a different part of that doc. There is no core team right now.

**Comment by directionless** [comment]
I'm going to merge this. despite the github teams URMs being kinda ugly

**Pull request merged**